yorm/tests/test_utilities.py

```diff
@@ -96,9 +96,71 @@ def it_requires_a_mapped_class_or_instance():
 
 def describe_match():
 
-    def it_is_not_yet_implemented():
-        with expect.raises(NotImplementedError):
-            utilities.match(Mock)
+    def class_factory(model_class, instance):
+        instance.__mapper__.create()
```

Comment:
I think these tests would help to prove functionality if they all saved multiple instances to disk and verified that only the matching instances are returned from `match()`.